### PR TITLE
Add oep data upload script

### LINKED ISSUE #255: Integrate OEP upload script into snakemake pipeline
The input and/or output data of oemof-B3 should be uploaded to the OEP automatically. Thus, the existing upload [script](https://github.com/OpenEnergyPlatform/tutorial/blob/564cab8127218a136ed265cf72ba03ca975746f3/upload/OEP_Upload_Process_Data_and_Metadata_oem2orm.ipynb) from the OEP tutorial section could be used.

### PR DESCRIPTION
**Pull request by chrwm**
Closes #255 

The script will enable an automated data upload to the OEP via the API.

In this version of the script, the credentials (OEP_USER & OEP_TOKEN) need to be saved in the environment.
The tables on the OEP are created from the metadata and later filled with the content from the csv-files. Thus, metadata (description of columns) for all tables need be in place before uploading the data itself.

### CODE REVIEW

File: scripts/data_oep_upload.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+The example file for this tutorial ('upload_tutorial_example_data.csv') is in the 'data' directory, next to this tutorial. Make sure to adapt the path to the file you're using if your file is located elsewhere.
+'''
+
+filepath = "./metadata/"
+
+list_filenames = os.listdir(filepath)
+
+files = [filename.split(".")[0] for filename in list_filenames]
+
+print('This files will be uploaded \n')
```
Comment (jnnr) [inline scripts/data_oep_upload.py:43]: The scripts in oemof-B3 use logging, not print. Can we adapt this?

**Review by jnnr — COMMENTED** [review]
Thanks for setting this up. As we agreed, the script should do this:
* Take as input the mapped results data of a specific scenario
* Upload the mapped results data to the oep.
* If necessary, update the data (?).

TODO: 
* [ ] There is no metadata for the results yet.
* [ ] Find out if it is possible to trigger the upload using snakemake.
* [ ] Give information on how to provide oep credentials in environment

File: scripts/data_oep_upload.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
+from oem2orm import oep_oedialect_oem2orm as oem2orm
```
[the remark this answers is not in the document]
Comment (chrwm) [reply scripts/data_oep_upload.py:1]: Sounds good to me.

[the remark this answers is not in the document]
Comment (chrwm) [reply scripts/data_oep_upload.py:1]: I have no preference regarding the point, but for consistency reasons in this repo and the script pipeline, it sounds reasonable.

File: scripts/data_oep_upload.py (hunk 1 above)
Comment (chrwm) [reply scripts/data_oep_upload.py:43]: Yes, from my side.

**Comment by chrwm** [comment]
@jnnr I changed the base, and merging into the repo-owned dev branch should enable editing the files.

**Review by jnnr — APPROVED** [review]

**Pull request merged**